__tests__/integration/create.test.ts

```diff
@@ -0,0 +1,138 @@
+import supertest from 'supertest';
+
+import { app } from '../../src/app';
+import httpStatus from 'http-status';
+import { DateTime } from 'luxon';
+
+const request = supertest(app);
+const ISO_DATE_REGEX = /^\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}.\d{3}Z$/;
```

Comment:
You may want to checkout js-joda: [docs](https://js-joda.github.io/js-joda/manual/ZonedDateTime.html#working-with-time-zones) | [npm](https://www.npmjs.com/package/@js-joda/core)

Reply:
OK

Reply:
Will surely have a look 👍